### ISSUE TITLE
MathJax Usage

### ISSUE THREAD

**Issue by samcarecho** [jtebert/jekyll-polymer#13]
The original code has a call to the MathJax CDN. It was shutdown.
https://www.mathjax.org/cdn-shutting-down/

What is it used for? Can we replace it for another library?

**Comment by jtebert** [comment]
MathJax is used for displaying equations (written in LaTeX format). MathJax is still in use, but they are no longer self-hosting a CDN. It's an easy update, though -- just replace the script source to `cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?...`

**Comment by samcarecho** [comment]
What if we make the source available as a part of the jekyll-polymer project, therefore removing the dependency on cdnjs?

-----------------

(Update)
I just checked and the source is available on GitHub.
https://github.com/mathjax/MathJax

Would it be ok to have it as a bower dependency, instead of using cdnjs?

**Comment by jtebert** [comment]
Yeah, I was going to suggest adding it as a locally installed dependency (bower/node/etc) ,rather than directly including the source directly. Definitely a preferable method to cdnjs if setup already requires Bower intsallations.
